json-schemas/interface-methods/permissions-grant.json

```diff
@@ -63,20 +63,7 @@
           "type": "string"
         },
         "scope": {
-          "oneOf": [
-            {
-              "$ref": "https://identity.foundation/dwn/json-schemas/permissions/scopes.json#/definitions/protocols-query-scope"
-            },
-            {
-              "$ref": "https://identity.foundation/dwn/json-schemas/permissions/scopes.json#/definitions/records-read-scope"
-            },
-            {
-              "$ref": "https://identity.foundation/dwn/json-schemas/permissions/scopes.json#/definitions/records-delete-scope"
-            },
-            {
-              "$ref": "https://identity.foundation/dwn/json-schemas/permissions/scopes.json#/definitions/records-write-scope"
-            }
-          ]
+          "$ref": "https://identity.foundation/dwn/json-schemas/permissions/defs.json#/definitions/scope"
```

Comment:
For my understanding: why is this no longer the `oneOf` definition?

Reply:
The `oneOf` got moved down a layer into `defs.json#/definitions/scope`. So it's still being used, just not in this file

Reply:
Got it, would be nice if `defs.json` was named the same as `permissions-definitions.json` for consistency, but won't block the PR for it.